QUEST_LV_0200.tsv

```diff
@@ -3179,44 +3179,44 @@ QUEST_LV_0200_20150323_003178	{memo X}The occurence of Stone Frosts became irreg
+QUEST_LV_0200_20150323_003188	$My hunger doesn't go away even after eating a lot.{nl}It's actually getting worse, I might as well just die.
 QUEST_LV_0200_20150323_003189	$There must be something to eat around here..{nl}Please.. get me something to eat.{nl}
-QUEST_LV_0200_20150323_003190	I am so hungry and something is echoing inside my head..{nl}The sound of a music box..  I can't eat an echo..
-QUEST_LV_0200_20150323_003191	Ahah, my Lily..{nl}I can probably never see you again.
-QUEST_LV_0200_20150323_003192	Please think of that necklace as the gift from Liliya and keep it safely.{nl}It is blessed from the great cathedral so it will help you.
-QUEST_LV_0200_20150323_003193	I am still hungry. Hungry.{nl}I can't endure anymore. I am hungry!
-QUEST_LV_0200_20150323_003194	I am so.. hungry.{nl}Hey, if you have any leftover food, if you could share it with me..
-QUEST_LV_0200_20150323_003195	If that's all you have, then please share it with me.. please..
-QUEST_LV_0200_20150323_003196	Meat..quickly. {nl}I feel like I am dying.
-QUEST_LV_0200_20150323_003197	Meat.. Meat!{nl}Please give it to me fast. Fast!
-QUEST_LV_0200_20150323_003198	Wait, that music box.. what was it?{nl}
-QUEST_LV_0200_20150323_003199	My memories.. she..{nl}Give it to me! That's mine!
-QUEST_LV_0200_20150323_003200	{memo X}Damn. What was I doing?{nl}Ahah. Liliya.. My Liliya!{nl}
-QUEST_LV_0200_20150323_003201	We left to the great cathedral for our pilgrimage to receive the blessing of the Goddesses to celebrate our wedding.{nl}But then, that evil Naktis.. that unforgivable Demon Lord.{nl}
+QUEST_LV_0200_20150323_003190	$I am so hungry yet something is echoing inside my head..{nl}The sound of a music box..  I can't eat an echo..
+QUEST_LV_0200_20150323_003191	$Ahah, my Lylia..{nl}I probably can never see you again.
+QUEST_LV_0200_20150323_003192	$Please think of that necklace as the gift from Lylia and keep it safely.{nl}It is blessed from the great cathedral so it will help you.
+QUEST_LV_0200_20150323_003193	$I am still hungry. Hungry.{nl}I can't take it anymore. I am hungry!
+QUEST_LV_0200_20150323_003194	$I am so.. hungry.{nl}Hey, if you have any leftover food, if you could share it with me..
+QUEST_LV_0200_20150323_003195	$Even if that's all you have, please share it with me.. please..
+QUEST_LV_0200_20150323_003196	$Meat..quickly. {nl}I feel like I am dying.
+QUEST_LV_0200_20150323_003197	$Meat.. Meat!{nl}Please give it to me fast. Fast!
+QUEST_LV_0200_20150323_003198	$Wait, that music box.. what was it?{nl}
+QUEST_LV_0200_20150323_003199	$My memories.. she..{nl}Give it to me! That's mine!
+QUEST_LV_0200_20150323_003200	{memo X}Damn. What was I doing?{nl}Ahah. Lylia.. My Lylia!{nl}
+QUEST_LV_0200_20150323_003201	$We left for a pilgrimage to the great cathedral to receive the blessing of the Goddesses to celebrate our wedding.{nl}But then, that evil Naktis.. that unforgivable Demon Lord.{nl}
```

Comment:
Great Cathedral?
Goddesses of Goddess? If Goddesses, blessings. If Goddess, blessing